agents-manage-ui/src/components/logo.tsx

```diff
@@ -1,48 +0,0 @@
-import { cn } from '@/lib/utils';
-
-export const Logo = (props: React.SVGProps<SVGSVGElement>) => {
-  return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      width="890"
-      height="169"
-      viewBox="0 0 890 169"
-      fill="none"
-      role="img"
-      aria-labelledby="inkeep-logo"
```

Comment:
removed this prop since:
<img width="451" height="510" alt="image" src="https://github.com/user-attachments/assets/4248ed34-0e26-4119-afee-26c7eb9076e5" />